wire-ios-transport/Tests/Source/URLSession/ServerCertificateTrustTests.swift

```diff
@@ -202,4 +208,6 @@ class BackendTrustProviderTests: XCTestCase {
         let host = "www.youtube.com"
         XCTAssertFalse(sut.verifyServerTrust(trust: serverTrust, host: host), "\(host) should NOT be trusted")
     }
+
+    // TODO: we should have tests for the certificate validity here as well
```

Comment:
question: also still todo or do we need a ticket?

Reply:
Added tests.